include/common_structs.h

```diff
@@ -4,6 +4,12 @@
 #include "ultra64.h"
 #include "types.h"
 
+typedef struct vec3f {
+    /* 0x00 */ f32 x;
+    /* 0x04 */ f32 y;
+    /* 0x08 */ f32 z;
+} vec3f; // size = 0x0C
```

Comment:
I'd personally prefer Vec3f; will see how Ethan feels about that.

Reply:
The capital version is what oot and co use, but we have a tendency in this project to use lowercase type names already (which I'm not a huge fan of, tbh, but I think that's a bigger conversation). I'd say we should use the lowercase version and then have another conversation about uppercase types later

Reply:
> which I'm not a huge fan of, tbh, but I think that's a bigger conversation
New issue incoming? I'm not a big fan of it either - Star Rod is even internally inconsistent.

For now, yeah, let's stay with lowercase.